runelite-client/src/main/java/net/runelite/client/plugins/skillcalculator/skills/RunecraftAction.java

```diff
@@ -74,4 +74,10 @@ public enum RunecraftAction implements ItemSkillAction
 	private final int level;
 	private final float xp;
 	private final boolean ignoreBonus;
+
+	@Override
+	public boolean isBonusApplicable(SkillBonus bonus)
```

Comment:
Not related to this PR, but I noticed that mind, body, and chaos cores incorrectly do not have `ignoreBonus = true`. Can you add a commit which fixes that as well? Thanks.